### DISCUSSION TITLE
Table sorting

### DISCUSSION THREAD

**Discussion by mariusa** [cdmoro/bootstrap-vue-3#707] [Ideas]
Glad to see progress on Table sorting!

Is it possible to let users sort by any column that is marked as sortable? 
https://github.com/cdmoro/bootstrap-vue-3/commit/0639fcea30017b6576feccdc20466344a399e149

Please also consider that some columns (shown as HTML, for example), might have an internal numeric index to sort by.
eg Date of birth, shown as a localized / translated string (not alphabetical), could have an `age` attribute attached, numeric, used for sorting (but not shown visually).

**Comment by jonkad** [comment]
Also it would be nice if there was an option for non case sensitive sorting.

Currently:
Aa
Cc
bb
zz

Should be:
Aa
bb
Cc
zz
